autopilot/ipfilter.go

```diff
@@ -0,0 +1,93 @@
+package autopilot
+
+import (
+	"fmt"
+	"net"
+)
+
+const (
+	// number of unique bits the host IP must have to prevent it from being filtered
+	IPv4FilterRange = 24
+	IPv6FilterRange = 54
+)
+
+type resolver interface {
+	LookupIP(string) ([]net.IP, error)
```

Comment:
if you change this to `LookupIP(ctx context.Context, network, host string) ([]net.IP, error)`, you can use `&net.Resolver{}` instead of defining `prodResolver`

Reply:
thanks!